### PR TITLE
Update to new YAML syntax

### PR DESCRIPTION
**Pull request by tdutrion**

### CODE REVIEW

File: workflows/push.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+on: push
+name: Main
+jobs:
+  pHP-CS-Fixer:
```
Comment (OskarStark) [inline workflows/push.yml]:
```suggestion
  PHP-CS-Fixer:
```

**Review by OskarStark — CHANGES_REQUESTED** [review]
Thank you!
Please apply my suggestion.

Could you also take a look at: https://github.com/OskarStark/phpstan-ga

File: workflows/push.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+on: push
```
Comment (OskarStark) [inline workflows/push.yml]: I am wondering, don't need for the `.github` folder? 🤔

File: workflows/push.yml (hunk 1 above)
Comment (tdutrion) [reply workflows/push.yml]: The Github migration tool did that, so I'm lowering it all (I guess that key should start lowercase, the uppercase name is described after)

File: workflows/push.yml (hunk 2 above)
Comment (tdutrion) [reply workflows/push.yml]: It should have been there, fixed, sorry :p

**Review by OskarStark — APPROVED** [review]
Thank you ❤️

**Comment by OskarStark** [comment]
Hmm, after this merge and a rebase, the 2 test PR's does not trigger the GithubAction anymore... 🤔

**Comment by tdutrion** [comment]
From the actions tab, we can see
>Workflows defined in main.workflow

Is there a setting in the repository settings or something?

**Comment by OskarStark** [comment]
I activated the new Workflow:
Screenshot 2019-08-14 09 18 57

**Comment by OskarStark** [comment]
Could you please tell me where I can convert the old to new files?

**Comment by tdutrion** [comment]
The documentation is here https://help.github.com/en/articles/migrating-github-actions-from-hcl-syntax-to-yaml-syntax
I'll do a PR for https://github.com/OskarStark/phpstan-ga if you want.

Thanks for the feedback on how to activate the new actions.

**Pull request merged**